zebra/zebra_vrf.h

```diff
@@ -193,7 +193,7 @@ struct zebra_vrf {
 static inline vrf_id_t zvrf_id(struct zebra_vrf *zvrf)
 {
 	if (!zvrf || !zvrf->vrf)
-		return VRF_UNKNOWN;
+		return VRF_DEFAULT;
```

Comment:
What's the reasoning here?

Reply:
@qlyoung I suggested this change to unify the output everywhere. Somewhere it was printed as 0, somewhere as UINT_MAX.